salt/docker/remove-pre-reboot.sls

```diff
@@ -11,11 +11,3 @@
 
 /etc/docker/certs.d:
   file.absent
-
-/var/lib/docker:
```

Comment:
I'm sure that we don't want compromise all users containers and all data which they contains. This should definitely stay